Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV ZB_HOME=/usr/local/zeebe \
     ZEEBE_GATEWAY_HOST=0.0.0.0 \
     ZEEBE_STANDALONE_GATEWAY=false
 ENV PATH "${ZB_HOME}/bin:${PATH}"
-ENV JAVA_OPTS -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap
+ENV JAVA_OPTS -XX:MaxRAMPercentage:80.0
```

Comment:
I guess this should be `MaxRAMPercentage=80.0`, i.e. `=` instead of `:`?

Reply:
fixed, with my sincere apologies